packages/pharos/src/components/checkbox-group/pharos-checkbox-group.ts

```diff
@@ -81,16 +81,11 @@ export class PharosCheckboxGroup extends FormElement {
   }
 
   protected override render(): TemplateResult {
-    const labels = ['legend'];
-    if (this.messageId) {
-      labels.push(this.messageId);
-    }
-
     return html`
       <fieldset
         class="checkbox-group checkbox-group--${this.horizontal ? 'horizontal' : 'vertical'}"
         aria-invalid="${this.invalidated}"
-        aria-labelledby="${labels.join(' ')}"
+        aria-describedby="message"
```

Comment:
As far as I can tell, it looks like it does only supply a value if there is a message, so that's promising. We may need to review some additional use cases to be sure.

Reply:
🤔 [this file](https://github.com/sirrah-tam/pharos/blob/5a7e6107e5ea2e47929d1c2b5b2053ccb46864f7/packages/pharos/src/components/base/form-element.ts#L104) seems to be where the messageId is populated from. The logic here is a little convoluted and I would have to look into it a it more to be sure.